src/Common/Configuration.lua

```diff
@@ -48,6 +53,12 @@ function Configuration.new(ConfigurationTable: {[string]: any}): Types.Configura
     self.BannedUsers = ConfigurationTable.BannedUsers or {}
     self.CommandLevelOverrides = ConfigurationTable.CommandLevelOverrides or {}
     self.FeatureFlagOverrides = ConfigurationTable.FeatureFlagOverrides or {}
+    self.CommandConfigurations = ConfigurationTable.CommandConfigurations or DEFAULT_COMMAND_CONFIGURATIONS
+    
+    -- Confirm all DEFAULT_COMMAND_CONFIGURATIONS exist within CommandConfigurations
+    for Name, Value in DEFAULT_COMMAND_CONFIGURATIONS do
+        self.CommandConfigurations[Name] = self.CommandConfigurations[Name] or Value
```

Comment:
Right now, this would work, but if a value came around that was a bool, this would break. A default of `true` when the user has it set to `false` would override it back to `true`. Suggestion:

```lua
for Name, Value in DEFAULT_COMMAND_CONFIGURATIONS do
    if self.CommandConfigurations[Name] ~= nil then continue end
    self.CommandConfigurations[Name] = self.CommandConfigurations[Name]
end
```